### PR TITLE
Ui performance

### PR DESCRIPTION
**Pull request by BrianAdams**
* Added simple loading animation 
* Optimized load time of the elements
* Added two caches. Use env "IGNORE_CACHE" to turn the cache off (for development).  When the cache is on, you are always served stale information very quickly the first time. Latest data on the second refresh.
    - System serves the browser pages from cache and then refreshes them 
    - The system caches the plugin folder contents after first load
* Cleaned up noise log message
* Fixed off-line caching to fix replay feature
* The vscode settings in this project now default to turn caching off, be sure to change the IGNORE_CACHE when testing.

### CODE REVIEW

**Comment by BrianAdams** [comment]
Test plan:
- [ ] Validate on BBB

**Comment by spiderkeys** [comment]
Definitely seems to be working faster (testing from mock mode). There are a few style bugs that happen in Chrome for me regarding the loading screen:

- The loading screen gets resized and you see a flicker of black border on the top and left sides
- A scrollbar appears on the right side briefly
- The text and other content of the page resizes to a smaller size
- At the beginning of the fade transition, it flickers from completely transparent back to opaque and then proceeds to fade out

**Comment by nerdgilbert** [comment]
Tested this branch on my system. Cockpit definitely seems to load faster. Nice! Worth noting that there is a strange flicker (resize, perhaps?) that is worth investigating further.

File: src/system-plugins/ui-manager/base.ejs [hunk 1]
```diff
@@ -98,23 +100,217 @@ ga('send', 'pageview');
...
+  <script>window.openrovtheme='<%=theme%>'</script>  
+  <link id="lazyScripts" rel="import" href="/all_scripts.html" async>
   <link rel="import" href="components/polymer/polymer.html">
-  <!-- Load the browser global services -->
-  <% for(var i=0; i<scripts.length; i++) {%>
-  <script type="text/javascript" src="<%= scripts[i] %>"></script><% } %>
+<style>
+    body.loading #splash {
+        opacity: 1;
+        z-index: 0;
```
Comment (nerdgilbert) [inline src/system-plugins/ui-manager/base.ejs]: I think this is where the black flicker is happening. Changing the z-index here to 9000 (or anything greater than cockpit's z-index) got rid of the flicker

**Pull request merged**